packages/core/src/shared/utilities/zipStream.ts

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import archiver from 'archiver'
+import { WritableStreamBuffer } from 'stream-buffers'
```

Comment:
Is stream-buffers usable in web browser (not nodejs)?

Reply:
I couldn't find much info, but I think it won't work natively in the browser without some additional configuration. Let me see if anything else is available as a follow up